### ISSUE TITLE
prop "features". Expected Collection, got Object 

### ISSUE THREAD

**Issue by akc4** [MelihAltintas/vue3-openlayers#6]
Thank you for adding the props and for creating a codesandbox.

Trying them out on I get the following warning:

```
[Vue warn]: Invalid prop: type check failed for prop "features". Expected Collection, got Object
```

I don't quite understand because I am declaring the `selectedFeature` as a `new Collection()` and then when selecting a feature, it add's it to the collection array.

Here is the codesandbox:
https://codesandbox.io/s/vue3-openlayers-forked-jftt9?file=/src/App.vue

**Comment by MelihAltintas** [comment]
I don't understand your purpose of using the features prop. You will use the features prop for the features that should be selected in the first mount. İf you want to get a selected features, you should use a @select event. 
`
        function featureSelected(event) {
             modifyEnabled.value = false
            if (event.selected.length > 0) {
                **selectedFeatures.value = event.target.getFeatures();**
                modifyEnabled.value = true
            }
        }
`

**Comment by MelihAltintas** [comment]
fixed by 0.1.26
